apps/graphql/src/apps/booking/dataloaders/SaveBooking.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import stringify from 'json-stable-stringify';
+import { OptimisticDataloader } from '@kiwicom/graphql-utils';
+
+import fetch from '../../../services/fetch/tequilaFetch';
+import type {
+  SaveBookingPayloadType,
+  SaveBookingOutputType,
+  Passenger,
+  HoldBags,
+} from '../Booking';
+
+export const parseParameters = (input: SaveBookingPayloadType) => {
+  return {
+    booking_token: input.bookingToken,
+    passengers: parsePassengers(input.passengers),
+    ...(input.lang && {
+      locale: input.lang.locale,
+      lang: input.lang.language,
+    }),
+    ...(input.paymentGateway ? { payment_gateway: input.paymentGateway } : {}),
+  };
+};
+
+export const parsePassengers = (passengers: Array<Passenger>): any => {
```

Comment:
Please use $ReadOnlyArray, and don't use return `any`

Reply:
Done!